reemgawad/theatr
# Teacher comment - edit & delete

**What changed:**
- Teacher can now edit or delete their comments

**How to test:**
- Login as a teacher and leave a comment on a student's answer, the comment should be displayed with an "edit" and "delete button underneath.
- Click on edit, should be able to edit and save. 
- Click on delete, should be able to delete the comment.
